src/entities/therapy-feedback.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from './base.entity';
+
+@Entity({ name: 'therapy_feedback' })
+export class TherapyFeedbackEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid', { name: 'therapyFeedbackId' })
+  id: string;
+
+  @Column({ unique: true })
+  bookingCode: string;
+
+  @Column()
+  email: string;
+
+  @Column({ type: Boolean, default: false })
```

Comment:
Sorry if we already spoke about this. This is a query re column with the name isFeedbackSent -  When would this be false? I would have thought the entry exists because feedback was sent?